Genius/Authenticator.cs

```diff
@@ -9,170 +9,161 @@
 
 namespace Genius
 {
```

Comment:
The whole file is shown as changed however nothing is 'actually changed' in the file. Please revert this file to the latest commit so that merging is clean.